controllers/platformnetwork_controller_fixtures.go

```diff
@@ -319,165 +315,165 @@ const DummyOAMUpdateResponse = `
 var HostsListBodyResponse string
 var SingleSystemBodyResponse string
 
-func HandleAddressPoolRequests(w http.ResponseWriter, r *http.Request) {
-	switch r.Method {
-	case http.MethodGet:
-		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprint(w, AddrPoolListBody)
-	case http.MethodPost:
-		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprint(w, DummyAddressPoolUpdateResponse)
-	case http.MethodPatch:
-		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprint(w, DummyAddressPoolUpdateResponse)
-	case http.MethodDelete:
-		w.WriteHeader(http.StatusNoContent)
-	default:
-		http.Error(w, `{"error": "Method not allowed"}`, http.StatusMethodNotAllowed)
-	}
-}
+// func HandleAddressPoolRequests(w http.ResponseWriter, r *http.Request) {
```

Comment:
Any consideration to comment these methods than delete them?

Reply:
For platformnetwork_controller most of it will not be applicable since reconciliation will be moved to platformnetwork sub-reconciler.
We can refer to these fixtures and methods step by step while writing UT for sub-reconciler which will be in a different file, that way we will have only what's required for sub-reconciler.